tgui/packages/tgui/interfaces/ExodroneConsole.tsx

```diff
@@ -0,0 +1,793 @@
+/* eslint-disable max-len */
```

Comment:
almost forgot to review this but you must once again face your max-len fears for this file too

i know it's a total slog but you'll find that the restriction only ends up restricting bad practices like cramming things on one line and not moving huge, deep segments to individual functions